### ISSUE TITLE
Logo/label

### ISSUE THREAD

**Issue by mikeapp** [ProjectMirador/mirador-design#3]
It would be helpful to see the logo represented (or did I miss it?).   Also, in addition to the `requireStatement` there is discussion of changing the `logo` to `provider` in order to allow a label.  This would provide a label with identity of the agent/institution providing the manifest, as well as the logo.  See https://github.com/IIIF/api/issues/1639

**Comment by regisrob** [comment]
I agree the Manifest `logo` should be clearly displayed (this is a requirement for many institutions and it is clearly stated as a MUST in the Presentation API spec): it should appear somewhere in the catalog (at least in the album preview?) and also in the information/metadata panel of the album window.

I feel like the question of "attribution" will be quite challenging for the Mirador 3 UI design, because if Mirador 3 is meant to support both P-2 and P-3 Manifests, it would be expected to handle the display of both the `requiredStatement` and `provider` properties (P-API v3) and the old `logo` and `attribution` from P-API v2.1...

**Comment by regisrob** [comment]
related to #1
